packages/jsapi-components/src/useBroadcastLoginListener.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  BroadcastLoginMessage,
+  BroadcastLogoutMessage,
+  isBroadcastLoginMessage,
+  isBroadcastLogoutMessage,
+  Message,
+} from '@deephaven/jsapi-utils';
```

Comment:
Yea - I agree, `jsapi-utils`, `jsapi-components`, and `jsapi-bootstrap` (perhaps jsapi-types as well) can likely all be combined into just `jsapi`. I'll leave that as a future change. The biggest difference right now is that `jsapi-utils` doesn't depend on React, and it depends on `jsapi-shim`. The boundary for jsapi-types makes sense on it's own, but it could just as easily be in the same package.
`jsapi-shim` should be deprecated, which we should do after @vbabich 's de-globalization change is ready to go.